package.json

```diff
@@ -20,11 +20,12 @@
   "author": "Augusto Lemble <me@augustolemble.com>",
   "license": "ISC",
   "dependencies": {
+    "abi-decoder": "^1.0.8",
     "ethereumjs-abi": "^0.6.4",
     "protobufjs": "^6.0.2",
     "solc": "^0.4.9",
     "truffle-hdwallet-provider": "0.0.3",
-    "zeppelin-solidity": "^1.0.4"
+    "zeppelin-solidity": "1.0.4"
```

Comment:
The way I see it, they are unrelated changes, and I like to use separate PRs for unrelated changes, except when there's a good reason to keep them together. Thinking that way, we could even have 3 separate PRs here: the change to `tx.origin`, fixing the zeppelin-solidity version, adding `approveData`.